index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
```suggestion
    <title>Der Ozean</title>
```
Ich glaube, dass "der Ozean" im header nicht im Browser angezeigt wird. Du könntest das als Titel nehmen und das h1 aus Zeile 13 in den header nehmen